tests/test_dagmc_stats.py

```diff
@@ -8,13 +8,19 @@
 import unittest
 
 test_input = "tests/3vols.h5m"
-
 my_core = core.Core()
 my_core.load_file(test_input)
-
 root_set = my_core.get_root_set()
+
 entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
 
+test_input_2 = "tests/single-cube.h5m"
```

Comment:
I always get a little uneasy when I see variable names with numbers in them to distinguish between cases.

A handy solution in python would be to have something like:
```
test_env = [ { 'input_file' : 'test/3vols.h5m'}, {'input_file' : 'tests/single-cube.h5m'} ]

for env in test_env:
     env['core'] = core.Core()
     env['core'].load_file(env['input_file'])
     env['root_set'] = env['core'].get_root_set()
     env['native_ranges'] = ds.get_native_ranges(env['core'], env['root_set'], entity_types)
     env['dagmc_tags'] = ds.get_dagmc_tags(env['core'])
```

Then in each test, you can select which `env` you want to use....